### DISCUSSION TITLE
I've just upgraded to 2.5, where can we update nvimtree options?

### DISCUSSION THREAD

**Discussion by Callumk7** [NvChad/NvChad#2851] [Q&A]
Hey, overall I am liking the changes, although I don't know where to override default settings for ui plugins that are installed by nvChad. in my previous config it was in custom/config/overrides M.nvimtree. 

Thanks in advance!

**Comment by siduck** [comment]
hi @Callumk7 , you can put your overrides anywhere and just import it in `opt` of your table where nvimtree is defined 

read https://nvchad.com/docs/config/plugins

**Reply by srirambtechit** [reply]
Hey @siduck , I want to override the key binding for `NvimTreeToggle` from `<C-n>` to something else as it conflicts with other plugin's key binding. Couldn't find a way around it? 

Solution I tried.
I updated the below code in `~/.config/nvim/lua/mappings.lua`

```lua
map("n", "<leader><C-n>", "<cmd>NvimTreeToggle<CR>", { desc = "nvimtree toggle window" })
```
but didn't work. Thanks.

P.S: I'm completely new the `neovim` world, It is a suggestion to add some sample content to override plugin settings, key bindings, event mapping etc, on the [official page docs.](https://nvchad.com/). It will be a benefit for a beginner like me.

**Reply by siduck** [reply]
@srirambtechit  your  code is valid, and it just sets C-n to nvimtree toggle,  so i suggest you to use `vim.keymap.del` to delete that mapping.

**Reply by srirambtechit** [reply]
Thanks @siduck. It worked.

**Reply by baiwfg2** [reply]
I don't use `vim.keymap.del` , it still works and the old mapping C-n still works.
